main.go

```diff
@@ -297,6 +297,18 @@ Generate an identity seed and launch a gateway:
 			EnvVars: []string{"RAINBOW_LIBP2P_LISTEN_ADDRS"},
 			Usage:   "Multiaddresses for libp2p bitswap client to listen on (comma-separated)",
 		},
+		&cli.StringFlag{
+			Name:    "tracing-auth",
+			Value:   "",
+			EnvVars: []string{"RAINBOW_TRACING_AUTH"},
+			Usage:   "If set the key gates use of the Traceparent header by requiring the key to be passed in the Authorization header",
+		},
+		&cli.Float64Flag{
+			Name:    "sampling-fraction",
+			Value:   0,
```

Comment:
Should this default to 1 instead of 0? I'd been using 0 with some testing to prevent flooding as I was more interested in Traceheaders, but the default OTEL sampler is "parentbased_always_on".

Reply:
Is it correct that setting to 1 would mean sample all, and setting to 0 would mean sample none? If that is the case, then `0.0` seems like a a good default. Would it make sense to use some pre-defined consts instead like `sampleHalf = 0.5`?